implementation/jobberknoll/app/use-cases/get-account-by-id-use-case.ts

```diff
@@ -11,24 +11,32 @@ import {
   type UUID,
 } from "@jobberknoll/core/shared";
 import type { AccountRepo } from "~/interfaces/mod.ts";
+import type { Logger } from "~/shared/mod.ts";
 import { UseCase } from "./use-case.ts";
 
+type GetAccountByIdReq = {
+  accountId: UUID;
+};
+
 export class GetAccountByIdUseCase extends UseCase<
```

Comment:
Moving from a plain `UUID` to `{ accountId: UUID }` to emphasise which ID it represents, since in the future there will be a couple UUIDs going around (account ID, user ID, request ID, ...).